### ISSUE TITLE
Update French translations

### ISSUE THREAD

**Issue by gregdenay** [squidfunk/mkdocs-material#4942]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### Language availability

- [X] I've checked the [list of available languages](https://bit.ly/33vFDD0)

### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "fr",
  "action.edit": "Editer cette page",
  "action.skip": "Aller au contenu",
  "action.view": "Consulter la source de cette page",
  "announce.dismiss": "Ne plus montrer cela",
  "blog.archive": "Archive",
  "blog.categories": "Catégories",
  "blog.categories.in": "dans",
  "blog.continue": "Continuer à lire",
  "blog.draft": "Brouillon",
  "blog.index": "Retourner à l'index",
  "blog.meta": "Metadonnées",
  "blog.references": "Liens connexes",
  "clipboard.copy": "Copier dans le presse-papier",
  "clipboard.copied": "Copié dans le presse-papier",
  "consent.accept": "Accepter",
  "consent.manage": "Paramétrer vos choix",
  "consent.reject": "Refuser",
  "footer": "Pied de page",
  "footer.next": "Suivant",
  "footer.previous": "Précédent",
  "header": "En-tête",
  "meta.comments": "Commentaires",
  "meta.source": "Source",
  "nav": "Navigation",
  "readtime.one": "1 min de lecture",
  "readtime.other": "# min lues",
  "rss.created": "Flux RSS",
  "rss.updated": "Flux RSS du contenu mis à jour",
  "search": "Recherche",
  "search.placeholder": "Rechercher",
  "search.share": "Partager",
  "search.reset": "Effacer",
  "search.result.initializer": "Initialisation de la recherche",
  "search.result.placeholder": "Taper pour démarrer la recherche",
  "search.result.none": "Aucun document trouvé",
  "search.result.one": "1 document trouvé",
  "search.result.other": "# documents trouvés",
  "search.result.more.one": "1 de plus sur cette page",
  "search.result.more.other": "# de plus sur cette page",
  "search.result.term.missing": "Non trouvé",
  "select.language": "Sélectionner la langue",
  "select.version": "Sélectionner la version",
  "source": "Aller au dépôt",
  "source.file.contributors": "Contributeurs",
  "source.file.date.created": "Créé",
  "source.file.date.updated": "Dernière mise à jour",
  "tabs": "Onglets",
  "toc": "Table des matières",
  "top": "Retour en haut de la page"
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Thanks for providing! We already received a new French translation in #4933 – can we merge then?

**Comment by squidfunk** [comment]
Added in 664af42367c696abf0d7f58e3903fb44579cde89.

**Comment by squidfunk** [comment]
Released as part of [9.0.8](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.8).
